docs/Getting Started/Recurring Tasks.md

```diff
@@ -78,7 +78,7 @@ Below is the same example as above, but this time the new task is scheduled base
 
 ```markdown
 - [ ] sweep the floors 🔁 every week when done ⏳ 2022-02-20
-- [x] sweep the floors 🔁 every week when done ⏳ 2021-02-06 ✅ 2022-02-13
+- [x] sweep the floors 🔁 every week when done ⏳ 2022-02-06 ✅ 2022-02-13
```

Comment:
Same as previous change; here the difference of including the `when done` is that the next scheduled date is February 20 (one week after completed date) rather than February 13 (one week after original scheduled date), but the year should still be the same.